feat(wizard): Add react wizard variants - [TET-774] (#6497)
getsentry/sentry-docs: src/wizard/javascript/react/with-error-monitoring-and-replay.md

---
name: React
doc_link: https://docs.sentry.io/platforms/javascript/guides/react/
support_level: production
type: framework

---

## Install
Sentry captures data by using an SDK within your application’s runtime.

```bash
# Using yarn
yarn add @sentry/react
```

## Configure
Initialize Sentry as early as possible in your application's lifecycle.

```javascript
import { createRoot } React from "react-dom/client";
import React from "react";  
import * as Sentry from "@sentry/react";
import App from "./App";

Sentry.init({
  dsn: "___PUBLIC_DSN___",

>   integrations: [new Sentry.Replay()],
>   // Session Replay
>   replaysSessionSampleRate: 0.1, // This sets the sample rate to be 10%. You may want this to be 100% while in development and sample at a lower rate in production
>   replaysOnErrorSampleRate: 1.0, // If the entire session is not sampled, use this sample rate to sample sessions when an error occurs.

Reviewer: ```suggestion
  integrations: [new Sentry.Replay()],
  // Session Replay
  replaysSessionSampleRate: 0.1, // This sets the sample rate at 10%. You may want to change it to 100% while in development and then sample at a lower rate in production.
  replaysOnErrorSampleRate: 1.0, // If you're not already sampling the entire session, change the sample rate to 100% when sampling sessions where errors occur.
```
